business/health.go

```diff
@@ -0,0 +1,53 @@
+package business
+
+import (
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/kubernetes"
+	"github.com/kiali/kiali/models"
+	"github.com/kiali/kiali/prometheus"
+	"k8s.io/api/apps/v1beta1"
```

Comment:
Trivial comment - all our kiali source code orders the imports with:

```
import (
  // std imports first (like "net/http", etc)
  // blank line
  // third-part imports (like "k8s.io" or "glog")
  // blank line
  // our kiali imports (like our "config")
```

The blank lines allow gofmt to organize each section. We just have to put them in that order so its easy for a person reading the code to find imports quickly with their eyes.

This is how all the kiali go code organizes the imports.

Reply:
Weird, I thought gofmt would take care of it for me, but it doesn't. Maybe I missed to configure something?